### DISCUSSION TITLE
Issue with Compiling (windows) (solved)

### DISCUSSION THREAD

**Discussion by VinnBee** [KadeArchive/Kade-Engine#777] [Q&A]
(Fixed) Decided just to upgrade to windows 10 and it worked,,,

I'm trying to compile or build kade engine on my windows 7 computer, I have made sure that I had everything installed, and to the correct versions, so I'm sure it can't be that I'm missing something, but whenever I start the building process it's stops here (see image below)
![Kade](https://user-images.githubusercontent.com/85513711/121065230-638daa00-c77d-11eb-85ce-470251667e12.PNG)
I'm not that good at this type of stuff but I'm pretty sure it shouldn't be stuck here for hours. I've also tried building the html5 version to see if I was missing anything and it didn't get stuck and built the game fine, sp I don't understand what's wrong with compiling the windows .exe version.

Is there any way I can fix this?

**Comment by ghost** [comment]
Hi - Windows 7 might not be supported. You can download Windows 10 media for free on Microsoft's website and get a free copy of Windows 10 without losing anything.

**Reply by puyoxyz** [reply]
Windows 7 is supported. This issue is unrelated to the OS version.

**Reply by ghost** [reply]
It literally requires win10 sdk

**Reply by PlayerPro342** [reply]
win 10 pro, or home?

**Reply by ghost** [reply]
Any really. Works fine for me on Home

**Reply by NotTheCoderGb** [reply]
> Windows 7 is supported. This issue is unrelated to the OS version.

get windows 10

**Comment by NotTheCoderGb** [comment]
Get windows 10, it literally needs windows 10 sdk

**Reply by ghost** [reply]
Exactly

**Reply by puyoxyz** [reply]
it needs the... version 10 of the windows sdk.. that doesn't necessarily mean windows 10

**Reply by ghost** [reply]
if not sp1 then vs wont run
